src/run.jl

```diff
@@ -543,5 +543,48 @@ function compute_linear_structure(parameters_file=nothing, source_terms=nothing;
 end
 
 
+"""
+    compute_jacobian_dg(parameters_file=nothing; verbose=false, refinement_level_increment=0, parameters...)
+
+Uses DG right hand side operator and simple second order finite difference to compute the Jacobian of the operator.
+The linearisation state is the initial condition from the parameter file.
+"""
+function compute_jacobian_dg(parameters_file=nothing; verbose=false, refinement_level_increment=0, parameters...)
```

Comment:
What happens if `parameters_file` is actually `nothing`? Does that even make sense?

Reply:
Nope, it doesn't. We just need to remove the default value `parameters_file=nothing` in `init_parameters`, `compute_linear_structure`, `compute_jacobian_dg`.

Reply:
this has been changed now